File: common/src/main/java/me/ftvkyo/fuzzy_broccoli/common/ArgumentProcessor.java
package me.ftvkyo.fuzzy_broccoli.common;

public class ArgumentProcessor {

    public static void printAll(String prefix, String[] args) {
        for(int i = 0; i < args.length; i++) {
            System.out.print(prefix);
            System.out.println("Argument[" + i + "]: " + args[i]);
        }
    }
}

File: common/src/main/java/me/ftvkyo/fuzzy_broccoli/common/ArgumentProcessor.kt
package me.ftvkyo.fuzzy_broccoli.common

object ArgumentProcessor {

    fun printAll(prefix: String, args: Array<String>) {
        for (i in args.indices) {
            print(prefix)
            println("Argument[" + i + "]: " + args[i])
        }
    }
}
